pkg/container-utils/containerd/containerd.go

```diff
@@ -261,12 +267,19 @@ func (c *ContainerdClient) taskAndContainerToContainerData(task *containerTask,
 	if err != nil {
 		return nil, fmt.Errorf("listing labels of container %q: %w", container.ID(), err)
 	}
+
+	image, err := container.Image(c.ctx)
+	if err != nil {
+		return nil, fmt.Errorf("getting image of container %q: %w", container.ID(), err)
+	}
+
 	containerData := &runtimeclient.ContainerData{
```

Comment:
I think we should combine code in `taskAndContainerToContainerData` and `GetContainer` to avoid having to update both of them every time we add a new field.

Reply:
Hi, Jose! That's a good idea. Thank you! 
I created an issue for this and I'll have it on my radar. WDYT?

Reply:
Tracked by https://github.com/inspektor-gadget/inspektor-gadget/issues/1911